src/entity/Categories.kt

```diff
@@ -0,0 +1,14 @@
+package entity
+
+enum class Categories {
+    Food,
+    Rent,
+    Salary,
+    Taxes,
+    HealthInsurance,
+    DentalInsurance,
+    Electricity,
+    Water,
+    Internet,
+    PhoneBill,
```

Comment:
Please follow the kotlin naming conventions for enum classes and use either UPPERCASE or SCREAMING_SNAKE_CASE as it mentioned [here](https://kotlinlang.org/docs/coding-conventions.html#property-names)

Reply:
@Mostafa-alsaygh